app/components/avo/index/resource_controls_component.html.erb

```diff
@@ -1,6 +1,9 @@
 <div class="space-x-2 flex flex-row justify-around w-full">
   <% button_classes = "text-gray-600 h-6 hover:text-gray-600" %>
-  <%= render Avo::Index::Ordering::ButtonsComponent.new resource: @resource, reflection: @reflection, view_type: @view_type %>
+
+  <% if can_reorder? %>
+    <%= render Avo::Index::Ordering::ButtonsComponent.new resource: @resource, reflection: @reflection, view_type: @view_type %>
+  <% end %>
```

Comment:
Could we move `can_reorder?` into the `render?` method of `Avo::Index::Ordering::ButtonsComponent`?
We already have some rendering logic in there.

```suggestion
  <%= render Avo::Index::Ordering::ButtonsComponent.new resource: @resource, reflection: @reflection, view_type: @view_type %>
```

Reply:
Totally doable I think.
I saw we also have render logic there, but here there are the others `can_some_action?` calls, and makes more sense to me to share this space.

I will move it when I have some time to write the tests. Thank you!

Reply:
@adrianthedev I tried adding `can_reorder?` into the `render?` by moving the whole method into `Avo::Index::Ordering::ButtonsComponent` but cannot make it work.
I believe I don't have access to `authorize_association_for` there and I'm unable to try a way to access it now.

Feel free to update it if you have the time or merge it as is if you want.
Otherwise, I will try to fix it on the weekend so maybe this can be included on the next release.
🙌

Reply:
If it can't be added, then it's fine if it's outside. You can leave the conditional there.

Reply:
@adrianthedev I'm sorry, I'm not going to be able to work on this soon, feel free to merge it as it is if you consider it appropriate.

Reply:
The only thing it's missing is tests.

Reply:
@adrianthedev Thanks for the help!

Reply:
No, no... Thank you for the help 💪😅